plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/index.tsx

```diff
@@ -322,6 +328,28 @@ const DatasetCard: React.FC<Props> = ({
   drag(dragRef);
   drop(preview(previewRef));
 
+  const downloadData = async (url: string, name?: string) => {
```

Comment:
useCallback
handleDataDownload

Reply:
Done!

Reply:
Sorry can you update the name of the callback too?

Reply:
Done!